src/OSPSuite.UI/Extensions/LayoutControlItemExtensions.cs

```diff
@@ -65,23 +73,18 @@ public static void AdjustControlSize(this LayoutControlItem layoutControlItem, i
          layoutControlItem.SizeConstraintsType = SizeConstraintsType.Custom;
          layoutControlItem.ControlMaxSize = size;
          layoutControlItem.ControlMinSize = size;
+         //In case the control size became smaller that the previous min size, the max size is ignored. So we set it again 
+         layoutControlItem.ControlMaxSize = size;
```

Comment:
This is the fix. If shrinking the control, ControlMinSize may be bigger than ControlMaxSize and the value is not accepted . Could be a new behavior from DevExpress...